packages/react-reconciler/src/ReactFiberHooks.new.js

```diff
@@ -2798,7 +2836,7 @@ const HooksDispatcherOnRerender: Dispatcher = {
   useDeferredValue: rerenderDeferredValue,
   useTransition: rerenderTransition,
   useMutableSource: updateMutableSource,
-  useSyncExternalStore: updateSyncExternalStore,
+  useSyncExternalStore: rerenderSyncExternalStore,
```

Comment:
Need to do this same change for the other "on rerender" dispatchers

Reply:
@#